criu/pie/restorer.c

```diff
@@ -425,6 +425,27 @@ static int restore_signals(siginfo_t *ptr, int nr, bool group)
 	return 0;
 }
 
+static int restore_rseq(struct rst_rseq_param *rseq)
+{
+	int ret;
+
+	if (!rseq->rseq_abi_pointer) {
+		pr_debug("rseq: nothing to restore\n");
```

Comment:
Should we handle the case when rseq is not supported as well?

Reply:
Sure. Fixed!